src/components/Locations.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import "../App.css";
+
+
+class Locations extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            location: []
+        }
+    }
+
+    fetchPeople = () => {
+        fetch('https://ghibliapi.herokuapp.com/location')
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                location: data,
+            })
+        })
+    }
```

Comment:
```suggestion
    fetchLocations = () => {
        fetch('https://ghibliapi.herokuapp.com/locations')
        .then((res) => res.json())
        .then((data) => {
            this.setState({
                locations: data,
            })
        })
    }
```
Your endpoint url isn't working because there's a typo in it. At the end of the url, it should say "location**s**" (with an "s"), not "location".
Your function is named `fetchPeople`, which isn't an appropriate name for a function that's meant to fetch data about a list of **locations**